apps/daffio/src/app/content/home/home-routing.module.ts

```diff
@@ -14,6 +14,19 @@ const homepage: Routes = [
     data: {
       title: 'Modern front-end development toolkit for ecommerce PWAs',
       description: 'Build your store with ease and flexibility — take advantage of the cutting edge with Angular (6+), Redux and Progressive Web Apps.',
+      daffSeoData: {
+        schema: {
+          "@context": "https://schema.org",
+          "@type": "Organization",
+          "url": "https://www.daff.io",
+          "sameAs": ["https://www.linkedin.com/company/daffodil-pwa/", "https://github.com/graycoreio/daffodil"],
+          "logo": "https://www.daff.io/assets/appicons/icon-512x512.png",
+          "name": "Daffodil",
+          "description": "A Modern front-end development toolkit for ecommerce PWAs",
+          "email": "hello@graycore.io",
+          "telephone": "+1-513-394-7356"
+        }
+      }

```

Comment:
I think `@daffodil/seo` should export a "route with schema" model that types this structure. https://github.com/graycoreio/daffodil/blob/1f1f85482b0d5d5fbdc9c5f5ec5f06ad2ff022cc/libs/router/src/named-view/models/route.type.ts#L8-L12 gives an example of this